### PR TITLE
Ports Katmos: Explosive Decompression and Fast Gasses 2 Electric Boogalo

### PR DESCRIPTION
**Pull request by dwasint**
# About The Pull Request
Basically a port of https://github.com/BeeStation/BeeStation-Hornet/pull/5013 with alot of changes to get running with auxmos 1.0

NOTE FOR MAINTAINERS: this updates our Auxmos to use the more up to date one, for the major fixes done to monster and katmos specifically and just katmos is general

## Why It's Good For The Game
Explosive Decompression makes holes in the station more of an actual threat as when the hole is made you will be pulled towards space or where ever a large difference in pressure is

Fast Gasses make both bombs more destructive initially and easier to repair over the long haul. With fast gasses gone are the days of areas being depressurized for long stretches of time after the hull has already been fixed. Now gasses fill areas and drain from areas much faster. This means breached areas should be usable again in a much shorter time frame assuming people have a space suit and some metal.

Also has a bandaid fix for firelocks instantly closing when pryed open. This will be properly fixed when i can figure out what is causing the issue in the first place

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
Explosive Decompression

Animation

</details>

## Changelog

:cl:
add: atmos made fast once again
fix: airlocks closing in your face
tweak: re-enabled fastmos/monstermos

/:cl:

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,11 +30,11 @@ RUN git init \
 # Build auxmos
 FROM rust-build as auxmos
 RUN git init \
-    && git remote add origin https://github.com/BeeStation/auxmos \
+    && git remote add origin https://github.com/Monkestation/auxmos \
```
Comment (nednaZ) [inline Dockerfile:33]:
```suggestion
    && git remote add origin https://github.com/Putnam3145/auxmos \
```

File: tools/ci/install_auxmos.sh [hunk 2]
```diff
@@ -4,6 +4,6 @@ set -euo pipefail
 source dependencies.sh
 
 mkdir -p ~/.byond/bin
-wget -O ~/.byond/bin/libauxmos.so "https://github.com/BeeStation/auxmos/releases/download/${AUXMOS_VERSION}/libauxmos.so"
+wget -O ~/.byond/bin/libauxmos.so "https://github.com/Monkestation/auxmos/releases/download/${AUXMOS_VERSION}/libauxmos.so"
```
Comment (nednaZ) [inline tools/ci/install_auxmos.sh:7]:
```suggestion
wget -O ~/.byond/bin/libauxmos.so "https://github.com/Putnam3145/auxmos/releases/download/${AUXMOS_VERSION}/libauxmos.so"
```

File: tools/tgs4_scripts/PreCompile.sh [hunk 3]
```diff
@@ -73,7 +73,7 @@ apt-get install -y cmake build-essential gcc-multilib g++-multilib cmake wget
 # update auxmos
 if [ ! -d "auxmos" ]; then
 	echo "Cloning Auxmos..."
-	git clone https://github.com/BeeStation/auxmos
+	git clone https://github.com/Monkestation/auxmos
```
Comment (nednaZ) [inline tools/tgs4_scripts/PreCompile.sh:76]:
```suggestion
	git clone https://github.com/Putnam3145/auxmos
```

File: dependencies.sh [hunk 4]
```diff
@@ -11,7 +11,7 @@ export BYOND_MINOR=1583
 export RUST_VERSION=1.54.0
```
Comment (nednaZ) [inline dependencies.sh:11]:
```suggestion
export RUST_VERSION=1.62.0
```

**Comment by dwasint** [comment]
closed for now as it won't compile on the server and my compile causes issues

**Pull request closed without merging**